packages/next/src/useNextRouter.ts

```diff
@@ -0,0 +1,16 @@
+import { useRouter } from 'next/router.js';
+import { waitForRouterReady } from './util';
+
+interface Options {
+  suspense?: boolean;
+}
+
+export function useNextRouter(options: Options = { suspense: true }) {
```

Comment:
```suggestion
export function useNextRouter(options: Options = { suspense: false }) {
```

suspense option is a feature that wasn't in the original version, so I think I'll get BC when I go up from the old version. So I think I'd better leave the default option false!

Reply:
Oh wow, that was entirely my mistake. Thanks so much for catching and preventing this awful breaking change! I’m happy to hear any suggestions.